neurons/validator.py

```diff
@@ -371,15 +371,15 @@ async def create_jobs(self):
                 )
             except Exception as e:
                 bt.logging.error(f"Error in create_jobs: {e}")
-            await asyncio.sleep(self.config.neuron.update_interval)
+
+            await asyncio.sleep(60)
 
     async def update_jobs(self):
         while True:
             try:
+                bt.logging.info("Updating jobs.")
                 bt.logging.info(f"step({self.step}) block({self.block})")
 
-                await asyncio.sleep(self.config.neuron.update_interval)
```

Comment:
I think the sleep should be at the beginning of the `update_jobs` function it makes more sense as we would only want to update jobs after we created them. This way it will be updating jobs as we create them